src/clientdlg.cpp

```diff
@@ -115,64 +112,67 @@ CClientDlg::CClientDlg ( CClient*        pNCliP,
     sldAudioPan->setAccessibleName ( tr ( "Local audio input fader (left/right)" ) );
 
     // reverberation level
-    QString strAudReverb = "<b>" + tr ( "Reverberation Level" ) + ":</b> " +
-        tr ( "A reverberation effect can be applied to one local mono audio channel or to both "
+    QString strAudReverb = "<b>" + tr ( "Reverb effect" ) + ":</b> " +
+        tr ( "Reverb can be applied to one local mono audio channel or to both "
         "channels in stereo mode. The mono channel selection and the "
-        "reverberation level can be modified. If, e.g., "
-        "the microphone signal is fed into the right audio channel of the "
-        "sound card and a reverberation effect shall be applied, set the "
+        "reverb level can be modified. For example, if "
+        "a microphone signal is fed in to the right audio channel of the "
+        "sound card and a reverb effect needs to be applied, set the "
         "channel selector to right and move the fader upwards until the "
-        "desired reverberation level is reached." ) + "<br>" + tr (
-        "The reverberation effect requires significant CPU so it should "
-        "only be used on fast PCs. If the reverberation level fader is set to "
-        "minimum (which is the default setting), the reverberation effect is "
+        "desired reverb level is reached." ) + "<br>" + tr (
+        "Reverb requires significant CPU so it should "
```

Comment:
I think we can remove this. This is a relic from old times. With modern computers, the CPU requirement should be negligible.